File: app/src/main/java/com/sierra/vagabond/data/DataStore.java
package com.sierra.vagabond.data;

import com.sierra.vagabond.data.entities.RecreationalAreaList;

public class DataStore {

  private static RecreationalAreaList recreationalAreaList;

  public DataStore(RecreationalAreaList recreationalAreaList) {}

  public static RecreationalAreaList getRecreationalAreaList() {
    return recreationalAreaList;
  }

  public static synchronized void setRecreationalAreaList(RecreationalAreaList list) {
    recreationalAreaList = list;
  }
}

File: app/src/main/java/com/sierra/vagabond/data/DataStore.kt
package com.sierra.vagabond.data

import com.sierra.vagabond.data.entities.RecreationalAreaList

class DataStore(recreationalAreaList: RecreationalAreaList) {
    companion object {

        @set:Synchronized
        var recreationalAreaList: RecreationalAreaList? = null
    }
}
